data_hub_api/docmaps/provider.py

```diff
@@ -259,10 +259,17 @@ def get_participants_for_preprint_peer_reviewed_step(
 def get_single_actions_value_for_preprint_peer_reviewed_step(
     query_result_item: dict,
     hypothesis_id: str,
+    peer_review_suffix: str,
     annotation_created_timestamp: str,
     outputs_type: str
 ) -> dict:
     preprint_doi = query_result_item['preprint_doi']
+    elife_doi_and_version = (
```

Comment:
minor: maybe call it `elife_version_doi` because the result is a doi.

Reply:
Maybe these two fields could be extracted as a function which could then be used in a test?